test/data/validationData/detailedBlobValidationSpec.yaml

```diff
@@ -70,7 +70,8 @@ components:
           enum: ['Postman']
         companyNumber:
           type: number
-          exclusiveMinimum: 10000
```

Comment:
is this file from our side or was there before, if latter then why are we changing it?